README.md

```diff
@@ -63,6 +63,20 @@ coming soon
 
 # Working on Docs
 
+## Datadog Staff
+
+* Always branch off of master
+* Create a branch `slack_handle/your_feature` if you would like to create a preview site and run tests
```

Comment:
but i think its a good naming strategy and we should encourage it. It seems to be what most do anyway